### PR TITLE
adding `git clean -fdx` to `prepare-release`

### PR DESCRIPTION
**Pull request by kellyselden**
An additional step for safety
cc @stefanpenner

### CODE REVIEW

**Comment by kellyselden** [comment]
friendly ping @stefanpenner

File: bin/prepare-release [hunk 1]
```diff
@@ -3,6 +3,7 @@
 rm -rf node_modules && \
 rm -rf ./ember-cli-*.tgz && \
```
Comment (stefanpenner) [inline bin/prepare-release]: we can drop these first two lines, as the `git clean -fdx` should do the trick

Comment (kellyselden) [inline bin/prepare-release]: updated

**Comment by stefanpenner** [comment]
:+1:

**Pull request merged**